core/Tracker/Db/Mysqli.php

```diff
@@ -75,11 +94,22 @@ public function connect()
 
         $this->connection = mysqli_init();
 
+
+        if($this->enable_ssl){
+            $this->connection.ssl_init($this->ssl_key, $this->ssl_cert, $this->ssl_ca, $this->ssl_ca_path, $this->ssl_cipher);
```

Comment:
`ssl_init` isn't a method on `connection`, instead you have to call `mysqli_ssl_set()` like so:

```
mysqli_ssl_set($this->connection, $this->ssl_key, ... etc. ...);
```

Reply:
oops, definitely should have been `set`, not `init`...
Would `connection.ssl_set` work?
Or is the procedural way preferred?
http://php.net/manual/en/mysqli.ssl-set.php

Object oriented style
```
bool mysqli::ssl_set ( string $key , string $cert , string $ca , string $capath , string $cipher )
```
Procedural style
```
bool mysqli_ssl_set ( mysqli $link , string $key , string $cert , string $ca , string $capath , string $cipher )
```

Reply:
So, in PHP, the `.` operator is actually the string concatenation operator. For member access, you'd have to use `->`.

Anyway, just tried it w/ `->ssl_set(`, and it works, so :+1: if you want to use that.